src/ui/page/file/view.jsx

```diff
@@ -305,10 +281,39 @@ class FilePage extends React.Component<Props> {
           </div>
 
           <div className="media__info--large">
+            <FileTags uri={uri} large />
             <FileDetails uri={uri} />
           </div>
         </div>
         <div className="grid-area--related">
+          <div className="media__uri-wrapper">
+            <Button
+              className="media__uri"
+              button="alt"
+              label={uri}
+              onClick={() => {
+                clipboard.writeText(uri);
+                showToast({
+                  message: __('Copied'),
+                });
+              }}
+            />
+            <div className="file-properties">
+              {isRewardContent && (
+                <Icon
+                  size={20}
+                  iconColor="red"
+                  icon={icons.FEATURED}
+                  // Figure out how to get the tooltip to overlap the navbar on the file page and I will love you
```

Comment:
I'm pretty sure I can explain/fix this, but easier to do so synchronously. Please bring up on next standup we're both on.